napari/layers/points/_points_mouse_bindings.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+
+from ._points_utils import points_in_box
+
+
+def select(layer, event):
+    """Select points.
+
+    Clicking on a point will select that point. If holding shift while clicking
+    that point will be added to or removed from the existing selection
+    depending on whether it is selected or not.
+
+    Clicking and dragging a point that is already selected will drag all the
+    currently selected points.
+
+    Clicking and dragging on an empty part of the canvas (i.e. not on a point)
+    will create a drag box that will select all points inside it when finished.
+    Holding shift throughout the entirety of this process will add those points
+    to any existing selection, otherwise these will become the only selected
+    points.
+    """
+    # on press
+    modify_selection = (
+        'Shift' in event.modifiers or 'Control' in event.modifiers
+    )
+
+    # if shift add / remove any from existing selection
+    if modify_selection:
+        # layer._value is defined in the base layer and contains the value
+        # under the cursor. For points, this is the index of the highlighted
+        # point.
+        if layer._value is not None:
+            if layer._value in layer.selected_data:
+                layer.selected_data.remove(layer._value)
+            else:
+                layer.selected_data.add(layer._value)
+            layer.selected_data = layer._selected_data
+        else:
+            pass
+    else:
+        if layer._value is not None:
+            if layer._value not in layer.selected_data:
```

Comment:
Can we remove this if? I think if I click without shift I want to select that point and that point only. (This is what happens in Keynote at least.)

Reply:
This is needed if you want to drag many points that are currently selected. If you take it away then you can only ever drag one at a time. It makes sense to me to leave, but I could drop if I understood the behavior we're trying to avoid better. Again this is something that could wait for more detailed usability testing

Reply:
I have added a comment to this effect

Reply:
Great, thanks! I was thinking about click, not drag.